README.md

```diff
@@ -296,68 +487,64 @@ docker run -v %cd%:/schematic \
   -c config.yml validate -mp tests/data/mock_manifests/inValid_Test_Manifest.csv -dt MockComponent -js /schematic/data/example.model.jsonld
 ```
 
-# Other Contribution Guidelines
+# Contribution Guidelines
+### Development process instruction
+
+For new features, bugs, enhancements:
+
+#### 1. Branch Setup
+* Pull the latest code from the develop branch in the upstream repository.
+* Checkout a new branch formatted like so: `develop-<feature/fix-name>` from the develop branch
+
+#### 2. Development Workflow
+* Develop on your new branch.
+* Ensure pyproject.toml and poetry.lock files are compatible with your environment.
+* Add changed files for tracking and commit changes using [best practices](https://www.perforce.com/blog/vcs/git-best-practices-git-commit)
+* Have granular commits: not “too many” file changes, and not hundreds of code lines of changes
+* You can choose to create a draft PR if you prefer to develop this way
+
+#### 3. Branch Management
+* Push code to `develop-<feature/fix-name>` in upstream repo:
+  ```
+  git push <upstream> develop-<feature/fix-name>
+  ```
+* Branch off `develop-<feature/fix-name>` if you need to work on multiple features associated with the same code base
+* After feature work is complete and before creating a PR to the develop branch in upstream
+    a. ensure that code runs locally
+    b. test for logical correctness locally
+    c. wait for git workflow to complete (e.g. tests are run) on github
+
+#### 4. Pull Request and Review
+* Create a PR from `develop-<feature/fix-name>` into the develop branch of the upstream repo
+* Request a code review on the PR
+* Once code is approved merge in the develop branch. We recommend squashing your commits for a cleaner commit history.
+* Once the actions pass on the main branch, delete the `develop-<feature/fix-name>` branch
+  
```

Comment:
Should this information live in the `CONTRIBUTION.md` file instead?

Reply:
Hi @BryanFauble good point - I've consolidated this section with what's in `CONTRIBUTION.md` and moved everything other in[ this commit](https://github.com/Sage-Bionetworks/schematic/pull/1510/commits/c3d45527dbc91374f14b31f710021120f9512cfa).

This is ready for a final review. Thanks!